src/ParametersHandler/include/BipedalLocomotion/ParametersHandler/IParametersHandler.h

```diff
@@ -192,6 +193,18 @@ class IParametersHandler
      */
     virtual std::string toString() const = 0;
 
+    /**
+     * Return the names of all the stored parameters.
+     * @return A set containing the names of all stored parameters.
+     */
+    virtual std::unordered_set<std::string> getParameterNames() const = 0;
+
+    /**
+     * Return the names of all the stored groups.
+     * @return A set containing the names of all stored groups.
+     */
+    virtual std::unordered_set<std::string> getGroupNames() const = 0;
```

Comment:
Just to understand is there a reason why we need these methods?

Reply:
I think that providing this interface some introspection, to some extent, is very useful. If you get an object without knowing the parameters and groups it contains, without the new `*Names` method is impossible to get the content. Even with them, you have to check all the possible types until you get the right `get*`. Quite hacky, but it works.

Anyway, as discussed with @S-Dafarra I'm going to move this change to a new PR, we can discuss details there.